src/main/java/com/google/sps/servlets/Util.java

```diff
@@ -53,6 +53,8 @@
 
 class Util {
 
+  private final String TOKEN_END_POINT = "https://oauth2.googleapis.com/token";
+  private final String REROUTE_LINK = "http://localhost:8080";
```

Comment:
Though now we might not want to deploy it, at least we need to add a comment or TODO here saying it only works locally now.